docs/admin/admission-controllers.md

```diff
@@ -142,6 +143,15 @@ applies a 0.1 CPU requirement to all Pods in the `default` namespace.
 
 See the [limitRange design doc](../design/admission_control_limit_range.md) and the [example of Limit Range](limitrange/) for more details.
 
+### InitialResources (experimental)
+
+This plug-in observes pod creation requests. For each container that does not specify compute resource requests or limits,
+the plug-in auto-populates a compute resource request based on historical usage of containers running the same image.
+If there is not enough data to make a decision the Request is left unchanged.
+When the plug-in sets a compute resource request, it annotates the pod with information on what compute resources it auto-populated.
```

Comment:
> In context of this document information about how to enabled it is redundant.

Can you explain what you mean by this? My worry is that people will see this section and assume this behavior is always on. It would be useful to tell them how to know whether it's on, how to enable/disable it, etc. It's fine to point to a section of a different document rather than creating a redundancy (e.g. say "see \<link\> for more information about how experimental features are enabled and disabled).